### PR TITLE
Connect execute button to endpoint

### PR DESCRIPTION
**Pull request by juliasaveliff**
- Also updated endpoint on backend to match other CP endpoints (with CP as query param)
- Fixed various typos

### CODE REVIEW

File: src/components/changePlanner/CPDetailView.tsx [hunk 1]
```diff
@@ -164,6 +164,23 @@ class CPDetailView extends React.Component<
...
+        getHeaders(this.props.token)
+      )
+      .then(res => {
+        this.addSuccessToast(res.data.success_message);
+      })
+      .catch(err => {
+        this.addErrorToast(err.response.data.failure_message);
+      });
+  }
+
```
Comment (juliasaveliff) [inline src/components/changePlanner/CPDetailView.tsx]: Added this method (all other changes are linting)

File: src/components/changePlanner/CPDetailView.tsx [hunk 2]
```diff
@@ -428,41 +445,42 @@ class CPDetailView extends React.Component<
...
+                <Callout title="No modifications for this change plan"> </Callout>
+              )}
           </ul>
         </div>
 
         <AnchorButton
           disabled={this.disableExecute()}
           icon="build"
           text="Execute Work Order"
+          onClick={() => this.handleExecute()}
```
Comment (juliasaveliff) [inline src/components/changePlanner/CPDetailView.tsx]: Added this button click (all other changes are linting)

File: src/components/elementView/detailedView/assetView/assetView.tsx [hunk 3]
```diff
@@ -484,7 +484,7 @@ export class AssetView extends React.PureComponent<
     )
       .then(res => {
         this.setState({ isDecommissionOpen: false });
-        this.addSuccessToast("Successfuly Decommissioned Asset");
+        this.addSuccessToast("Successfully Decommissioned Asset");
```
Comment (juliasaveliff) [inline src/components/elementView/detailedView/assetView/assetView.tsx]: Fixed this typo - all other changes in file are linting

File: rackcity/api/urls.py [hunk 4]
```diff
@@ -66,8 +66,8 @@
     path('change-plans/modify', views.change_plan_modify),
     path('change-plans/delete', views.change_plan_delete),
     path('change-plans/pages', views.change_plan_page_count),
-    path('change-plans/execute', views.change_plan_execute),
     path('change-plans/<int:id>', views.change_plan_detail),
-    path('change-plans/<int:id>/resolve-conflict', views.change_plan_resolve_conflict),
+    path('change-plans/<int:id>/execute', views.change_plan_execute),
```
Comment (juliasaveliff) [inline rackcity/api/urls.py]: Updated endpoint to have CP id as query param

File: src/components/changePlanner/CPDetailView.tsx (hunk 1 above)
Comment (clairecfu) [reply src/components/changePlanner/CPDetailView.tsx]: did you test this? I think for post requests, the default second argument is the body, and the third is the headers. if no body you can just pass in {}

Comment (juliasaveliff) [reply src/components/changePlanner/CPDetailView.tsx]: THAT EXPLAINS IT

**Pull request merged**